sagemaker-debugger/tensorflow_profiling/low_batch_size.ipynb

```diff
@@ -26,7 +26,7 @@
    "source": [
     "import sys\n",
     "import IPython\n",
-    "install_needed = True  # should only be True once\n",
+    "install_needed = False  # should only be True once\n",
```

Comment:
If it runs E2E in both Mead and Studio, should the section with the option to restart the kernel be removed entirely?

Reply:
I have thought about it and yes, that is possible. However, I thought it is still good to leave it in there just in case customers run into problems (i.e. they are using older container, etc). But I am fine with either keeping or removing it.